lib/features/caching/config/hive_cache_config.dart

```diff
@@ -31,6 +37,24 @@ class HiveCacheConfig {
     }
   }
 
+  static Future<Uint8List?> getEncryptionKey() async {
+    final sharedPreference = Get.find<SharedPreferences>();
+    final containsEncryptionKey = sharedPreference.containsKey(LoginConstant.keyHiveEncrypt);
+    if (!containsEncryptionKey) {
+      final key = Hive.generateSecureKey();
+      await sharedPreference.setString(LoginConstant.keyHiveEncrypt, base64UrlEncode(key));
```

Comment:
why not store `encryptionKey` in Hive also ?

Reply:
or use [flutter_secure_storage](https://pub.dev/packages/flutter_secure_storage)

Reply:
> or use [flutter_secure_storage](https://pub.dev/packages/flutter_secure_storage)

`flutter_secure_storage` requires some configuration for platforms like. The `Android` side is `minSdkVersion is 18`. I don't want to change it so I don't use it at this time.

Reply:
> why not store `encryptionKey` in Hive also ?

I will look into whether hive should be used.